styless.css

```
@@ -0,0 +1,233 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+* {
+    font-family: 'Poppins', sans-serif;
+}
+
+.header{
+    padding-left: 40px;
+    padding-right: 30px;
+    padding-bottom: 7px;
+}
+
+
+
+.secciones0{
+    display: flex;
+    flex-direction: row;
+    font-family: 'Poppins', sans-serif;
+    justify-content: space-between;
+    font-size: 12px;
+    font-weight: bold;
+    
+}
+
+.secciones{
+    display: flex;
+    flex-direction: row;
+    list-style-type: none;
+    color: black;
+    justify-content: space-between;
+  
+}
+
+.blackHeart{
+    height: 12px;
+    width: 25px;
+}
+
+.bh{
+    display: flex;
+    align-items: center;
+    
+}
+
+.inputdiv{
+    width: 1100px;
+}
+
+.input1{
+    background-color: white;
+    width: 1100px;
+    background-repeat: no-repeat;
+    border: none;
+    background-color: rgba(0,0,0,.05); 
+    color: rgba(0,0,0,.5);
+    height: 30px;
+}
+
+.npmbtns{
+    padding-right: 30px;
+}
+
+.Join{
+    background-color: white;
+    border-width: 1px;
+
+}
+
+.Search{
+    background-color: #fb3e44;
+    color: white;
+    border: none;
+    height: 30px;
+    width: 70px;
+    transform: translateX(-40px);
+    
+}
+
+.Login{
+    background-color: white;
+    border: none;
+}
+
+.li1{
+    padding-left: 10px;
+    text-decoration: none;   
+}
+
+.li2{
+    text-decoration: none;
+    color: black;
+}
+
+.secciones1{
+
+    text-decoration: none;
+}
+
+.npmbox{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
```

Esta propiedad de flexbox hace la separación mas pareja que  ya que la otra los junta a los extremos 
```suggestion
  justify-content: space-evenly;
```